hooks/build

```diff
@@ -0,0 +1,43 @@
+#!/bin/sh -e
+
+WORKDIR="$(pwd)"
+IMAGE_NAME=${IMAGE_NAME:-"maniator/dind-buildx"}
+DOCKERFILE_PATH=${DOCKERFILE_PATH:-"$WORKDIR/Dockerfile"}
+DOCKER_TAG=${DOCKER_TAG:-"latest"}
+DOCKER_CONFIG=${DOCKER_CONFIG:-"$WORKDIR/.docker"}
```

Comment:
Same thing with the quotes here, they are not needed.

Reply:
thanks! will update